themes/jquery/header.php

```diff
@@ -104,6 +104,11 @@
 	</section>
 </header>
 
+<div id="banner-blm">
+    <h2>#BlackLivesMatter</h2>
```

Comment:
Please use tabs instead of spaces here. :)

Reply:
Maybe we could add a `.editorconfig` to this repo. Spaces are my default as well.

Reply:
Not suggesting that for this PR, just a thought.

Reply:
should be fixed now!

Reply:
@timmywil yes, that's a good idea.